.cirrus.yml

```diff
@@ -0,0 +1,38 @@
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus
+
+task:
+  name: Test on Graviton2 ($CC)
+  arm_container:
+    image: ghcr.io/ruby/ruby-ci-image:$CC
+    cpu: 4
+    memory: 16G
+  env:
+    CIRRUS_CLONE_DEPTH: 0 # do a full clone since on a large history it's less CPU intense
+    RUBY_DEBUG: ci
+    NPROC: 4
```

Comment:
The `NPROC` need to be `NPROC: 8` as well. Is there way to set a value of `cpu:` here to avoid hardcored number?

Reply:
Ah no, we actually don't need `NPROC` in the `.cirrus.yml`. In the `.travis.yml`, the environment variable is used to set `-jN`. But in `.cirrus.yml`, the `JOBS: -j5` is hardcored. So, we don't need the `NPROC`.

.travis.yml

```
    - NPROC="`nproc`"
    # JOBS and SETARCH are overridden when necessary; see below.
    - JOBS=-j$((1+${NPROC}))
```